src/com/laba/discountpackage/Discount.java

```diff
@@ -0,0 +1,39 @@
+package com.laba.discountpackage;
+
+
+public class Discount implements DiscountInterface {
+
+    private static Boolean newUser= true;
```

Comment:
```suggestion
    private static Boolean newUser = true;
```
This is not a proper way to use static, in this case all discounts will be the same, either for new user or not, which is not correct.